dom.bs

```diff
@@ -1225,12 +1240,28 @@ for discussion).
    <li><p>Otherwise, set <var>target</var> to <var>parent</var> and append (<var>parent</var>,
    <var>target</var>, <var>relatedTarget</var>) to <var>event</var>'s <a for=Event>path</a>.
 
-   <li><p>If <var>parent</var> is non-null, then set <var>parent</var> to the result of invoking
-   <var>parent</var>'s <a>get the parent</a> with <var>event</var>.
+   <li>
+    <p>If <var>parent</var> is non-null, then:
+
+    <ol>
+     <li><p>If <var>event</var>'s {{Event/bubbles}} attribute is true, <var>activationTarget</var>
+     is null, and <var>parent</var> has <a>activation behavior</a>, then set
+     <var>activationTarget</var> to <var>parent</var>.
+     <!-- If an EventTarget object with activation behavior can ever have an associated shadow root
+          we need to complicate this somewhat. -->
```

Comment:
So the only reason this isn't complicated yet is that we disallow attachShadow on elements with interesting activation behavior? That is kind of fragile, if so... hmm.